src/prescriptions/prescription-details.component.tsx

```diff
@@ -59,7 +79,13 @@ const PrescriptionDetails: React.FC<{
       {requests &&
         requests.map((request) => {
           return (
-            <MedicationEventCard key={request.id} medicationEvent={request} />
+            <Tile className={styles.prescriptionTile}>
+              <MedicationEvent
```

Comment:
Same here... the Tile has been pulled out into the Prescription Details component so that custom left-hand strip can be applied.